packages/reporter/src/runnables/runnables-store.ts

```diff
@@ -200,7 +200,7 @@ export class RunnablesStore {
   }
 
   @action
-  setRunningSpec (specPath: string) {
+  setRunningSpec (specPath: string | null) {
```

Comment:
when the specPath is null, should we be storing the history?

Reply:
See comment: https://github.com/cypress-io/cypress/pull/20663/files/be5b95a3d923368e80fae240c7a2e0740be4da07#r832425950. I've reverted this change since I don't know the answer, but my understanding is spec can never be null (not sure how strictly this type safety is actually exercised in the entire system, but the current code works, so it's probably safest not to touch this, after all).